Mathlib/Topology/UnitInterval.lean

```diff
@@ -133,6 +141,62 @@ theorem strictAnti_symm : StrictAnti σ := fun _ _ h ↦ sub_lt_sub_left (α :=
 theorem half_le_symm_iff (t : I) : 1 / 2 ≤ (σ t : ℝ) ↔ (t : ℝ) ≤ 1 / 2 := by
   rw [coe_symm_eq, le_sub_iff_add_le, add_comm, ← le_sub_iff_add_le, sub_half]
 
+@[simp]
+lemma eq_zero_iff_sym_eq_one (i : I) : σ i = 1 ↔ i = 0 := by
+  apply Iff.intro
+  · intro h
+    rw [← unitInterval.symm_zero] at h
+    exact symm_bijective.injective h
+  · intro h
+    rw [h, symm_zero]
+
+@[simp]
+lemma eq_one_iff_sym_eq_zero (i : I) : σ i = 0 ↔ i = 1 := by
+  apply Iff.intro
+  · intro h
+    rw [← unitInterval.symm_one] at h
+    exact symm_bijective.injective h
+  · intro h
+    rw [h, symm_one]
+
+theorem le_symm_if_le_symm (i j : I) : i ≤ σ j → j ≤ σ i := by
+  intro h
```

Comment:
```suggestion
theorem le_symm_if_le_symm (i j : I) (h : i ≤ σ j) : j ≤ σ i := by
```
Though it seems like i and j are determined by h, so should be implicit as well

Also below